packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -643,6 +644,30 @@ where
 
         Ok(entity_id)
     }
+
+    async fn update_link_active(
+        &self,
+        active: bool,
+        source_entity: EntityId,
+        target_entity: EntityId,
+        link_type_version_id: VersionId,
+    ) -> Result<(), LinkActivationError> {
```

Comment:
The create-operation takes a `Link`. Is there a specific reason, why we don't use this type here?

Reply:
Good point, we could just construct the link in the tests. Will update

Reply:
This takes a version ID and not a version URL. I could do the lookup for the version_id here instead, but that means 2 lookups in create if we're reactivating a link

Reply:
I missed that, sorry. Then we probably want to stick with `VersionId` for now. 🙂